src/create-style.js

```diff
@@ -65,6 +65,8 @@ const radii = (R) => (val) => {
       return { borderRadius: rx(0, 0, R, R) }
     case 'left':
       return { borderRadius: rx(R, 0, 0, R) }
+    case 'circle':
+      return { borderRadius: '50%' }
```

Comment:
Why not use a large number here so that it looks like a pill shape rather than an ellipsis?